### PR TITLE
Add inefficient multiply() without type hints

### PR DESCRIPTION
**Pull request by AjaySankar306**

### CODE REVIEW

File: tests/test_mathops.py [hunk 1]
```diff
@@ -2,3 +2,4 @@
```
Comment (AjaySankar306) [inline tests/test_mathops.py:2]: Are there any edge cases you considered, like negative numbers or multiplying by zero? It might be helpful to add tests for those cases.

Comment (AjaySankar306) [reply tests/test_mathops.py:2]: no

**Review by AjaySankar306 — COMMENTED** [review]
Are there any edge cases you considered, like negative numbers or multiplying by zero? It might be helpful to add tests for those cases.
